### PR TITLE
services: dbx helper

### PR DESCRIPTION
**Pull request by lzap**
Adds context to most if not all service `db.DB` or `db.Org` calls. This ensures SQL logs are correlated too.

### CODE REVIEW

File: pkg/services/services.go [hunk 1]
```diff
@@ -26,3 +28,8 @@ func NewService(ctx context.Context, log log.FieldLogger) Service {
 func (s *Service) SetLog(log log.FieldLogger) {
 	s.log = log
 }
+
+// DB returns db.DBx with context stored in the service
+func (s *Service) DB() *gorm.DB {
+	return db.DBx(s.ctx)
+}
```
Comment (lzap) [inline pkg/services/services.go:35]: This is the helper that is used in this patch. The other `db.Orgx` was introduced earlier.

**Comment by lzap** [comment]
Rebased to use `DBx/Orgx` methods instead, this is a tad cleaner.

**Review by ezr-ondrej — APPROVED** [review]
:orange_heart: it  :+1:

**Comment by ezr-ondrej** [comment]
Eh the coverage is flaky, lets remove it and use snyk here instead: https://github.com/RedHatInsights/edge-api/pull/2555

**Review by ezr-ondrej — APPROVED** [review]
Thanks @lzap ! :orange_heart:

**Pull request merged**